tests/onnx/test_onnx_v2.py

```diff
@@ -179,82 +179,79 @@ def test_values_override(self):
 
 
 PYTORCH_EXPORT_MODELS = {
-    ("albert", "hf-internal-testing/tiny-albert"),
-    ("bert", "bert-base-cased"),
-    ("big-bird", "google/bigbird-roberta-base"),
-    ("ibert", "kssteven/ibert-roberta-base"),
+    ("albert", "hf-internal-testing/tiny-random-AlbertModel"),
+    ("bert", "hf-internal-testing/tiny-random-BertModel"),
+    ("beit", "microsoft/beit-base-patch16-224"),
```

Comment:
I too the liberty of re-arranging these tests alphabetically to make it easier to track